CommunityToolkit.Diagnostics/CommunityToolkit.Diagnostics.csproj

```diff
@@ -44,6 +44,12 @@
     </When>
   </Choose>
 
+  <!-- T4 service used by the Guard APIs -->
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+
+  <!-- Text Templates to generate sources -->
```

Comment:
```suggestion
  <!-- Text templates to generate sources -->
```

Reply:
Since the extension is `tt`, I capitalised the second word to emphasize that.

Reply:
I'm aware of that. As I mentioned, please, let's try to avoid back and forth for every single review comment.
This whole PR is already taking way too much time for something that I don't even think is that worth it to be honest.